ImageSegmentation/ExtractContourWithSnakes.cxx

```diff
@@ -103,7 +103,7 @@ int main( int argc, char* argv[] )
     {
     P = computeP(alpha, beta, gamma, N);
     }
-  catch (int n)
+  catch (...)
     {
     return EXIT_FAILURE;;
```

Comment:
Side-note: there is an extra semi-colon here

Reply:
Fixed once PR #14 was merged and having rebased on top of master: 3764fad.